index.js

```diff
@@ -439,6 +447,12 @@ Converter.prototype.buildParameter = function(oldParameter) {
       'Complex type is used inside non-body argument.');
   }
 
+  //According to Swagger 2.0 spec: If the parameter is in "path",
+  //this property is required and its value MUST be true.
+  if (parameter.in === 'path') {
```

Comment:
@mohsen1 - yes, you do. The schema also requires it - https://github.com/swagger-api/swagger-spec/blob/master/schemas/v2.0/schema.json#L774-L778.